apps/condo/index.js

```diff
@@ -143,6 +146,84 @@ const apps = () => {
 
 /** @type {(app: import('express').Application) => void} */
 const extendExpressApp = (app) => {
+    const runtimeStats = {
+        activeRequestsIds: new Set(),
+        activeRequestsCountByType: {
+            graphql: 0,
+            api: 0,
+            oidc: 0,
+            wellKnown: 0,
+            healthCheck: 0,
+            other: 0,
+        },
+        totalRequestsCount: 0,
+        totalRequestsByTarget: {},
+    }
+
+    app.use(function runtimeStatsMiddleware (req, res, next) {
+        try {
+            const url = new URL(`${conf['SERVER_URL']}${req.url}`).pathname
+            const method = (req.method || 'get').toLowerCase()
+            const xTargetHeader = req.headers['x-target'] || 'other'
```

Comment:
You should sanitise this!!!

Otherwise I can brute-force values and memory leak

Reply:
Done